tests/cypress/e2e/addUserToGroup.test.cy.ts

```diff
@@ -0,0 +1,47 @@
+import { createSite, deleteSite, createUser, deleteUser } from '@jahia/cypress'
+import { generateRandomID } from '../utils/utils'
+import { SiteSettingsGroups } from '../page-object/siteSettingsGroups'
+describe('Add user to group', () => {
+    const siteKey = 'siteSettingsSite'
+    const languages = ['en', 'fr', 'de']
+    const userNameTest = 'user1_' + generateRandomID()
+    const groupNameTest = 'group_' + generateRandomID()
+
+    before(function () {
+        createSite(siteKey, {
+            languages: languages.join(','),
+            templateSet: 'dx-base-demo-templates',
+            serverName: 'localhost',
+            locale: 'en',
+        })
+        cy.apollo({
```

Comment:
Do we need to call createContent.graphql in this test? I do not see where we use the contents in the tests